docs/authentication/saml/jit-provisioning.mdx

```diff
@@ -0,0 +1,25 @@
+---
+description: Learn about Clerk's support for Just-in-Time (JIT) Provisioning during SAML SSO.
+---
+
+# Just-in-Time (JIT) Provisioning during SAML SSO
+
+Just-in-Time (JIT) Provisioning, or automatic account provisioning, is a process by which accounts for employees are created on-demand during the first time they authenticate via SAML SSO in your application.
+
+Using JIT Provisioning means your IT department won't have to manually create user accounts for each of the services or apps your employees use to get work done.
+
+Clerk supports JIT account provisioning for all [supported SAML providers](/docs/authentication/saml/overview) out of the box. This feature is available for any Clerk app that has the **[Sync user attributes during Sign in](#sync-user-attributes-during-sign-in)** setting enabled, which is enabled by default.
```

Comment:
Customers can only control if they want the attributes to be synced in each sign in using the  `Sync user attributes during Sign in` option. They can't control the behavior during the initial sign-up, so it's better to drop this line as we have a dedicated section below for the sign in setting

```suggestion
Clerk supports JIT account provisioning for all [supported SAML providers](/docs/authentication/saml/overview) out of the box.
```